README.md

```diff
@@ -104,6 +104,32 @@ sw  $t3, 8($t0)    # the_list[i+1] = $t3
     * Trade off of efficiency and readability
     * Loss of reusability, increased chance of errors
     * Try to only optimise code which runs often, do not worry about code which runs very rarely
+* Simple comparison
```

Comment:
Just say "Comparison"
Move this up somewhere, shouldn't be after compiler optimisations because order is very unintuitive.